### ISSUE TITLE
Debian/Ubuntu package generation removed 

### ISSUE THREAD

**Issue by mittwinter** [sharelatex/sharelatex#292]
Hey,
I've installed sharelatex following the guide in this gist [0] on Ubuntu 14.04 which used the build:deb grunt task to generate an installable Debian/Ubuntu package.
[0] https://gist.github.com/moschlar/2b941f75a44fbac69993

However, when trying to upgrade to the latest version of sharelatex (0.1.4) following the same procedure and trying to build a .deb, I found the functionality to be no longer available. Apparently, it was removed by commit c96d713310b73cef82ee0a933d22d29aadb129b4 for being "unused/deprecated". But at least from the comments in the gist, one can see that it was definitely not unsused.
And for it being deprecated, I would not consider the official instructions [1] more preferrable, since e. g. I don't consider running npm install and grunt install with root privileges considerable, as for my understanding, installing anything system-wide is the domain of the official package manager and no other instance (e. g. to prevent collisions, etc.). Plus, the .deb package required a lot less work to set up, as e. g. the official guide requires configuring upstart services by hand (section "Running sharelatex") whereas all those files were correctly installed in-place by the .deb package.
[1] https://github.com/sharelatex/sharelatex/wiki/Production-Installation-Instructions

So to sum this up, I think it would be very much appreciated if you would bring the functionality removed by commit c96d713310b73cef82ee0a933d22d29aadb129b4 back.

**Comment by henryoswald** [comment]
Hi,

We decided to kill of the .deb package because only a subset of people were on a debian machine so it was not that useful. Instead we have moved to using docker images which should be super simple for you to get setup with and works on multiple platforms.

**Comment by T7o7heVV** [comment]
I am also stranded in the installation because of the removal of this patch. :-(

**Comment by henryoswald** [comment]
Please feel free to support this Deb package but we will be focusing on
docker going forward b

On Sunday, 3 January 2016, T7o7heVV <notifications@github.com> wrote:

> I am also stranded in the installation because of the removal of this
> patch. :-(
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sharelatex/sharelatex/issues/292#issuecomment-168538001>
> .
>


-- 
Henry Oswald
shareLaTeX.com co-founder

**Comment by condector** [comment]
Gosh... I don´t have a production environment to use with Docker.. that´s why I follow the same steps from the [1].

But if this is a decision by people of the project.. :(

[1] https://gist.github.com/moschlar/2b941f75a44fbac69993

**Comment by henryoswald** [comment]
sorry to disappoint, you can still run all the services individually but it is so complex we don't support it, hence why we are pushing for the ready to roll docker install.

**Comment by condector** [comment]
No problem at all Henry.

I saw right now that Xenserver 7.0 support Docker. I will se more about it. :) Thanks.
